.github/workflows/autoupdate.yml

```diff
@@ -0,0 +1,45 @@
+# Creates PR for updating RGBDS version automatcally
+# put RGBDS version as "version" parameter like "v0.9.1"
+name: Create PR for new RGBDS version
+on:
+  workflow_dispatch:
+    inputs:
+      version:
```

Comment:
version should be explain here with a comment. Also mention that version has to be a valid git tag from the releases (put link to rgbds/releases)